public/docs/ts/latest/tutorial/toh-pt6.jade

```diff
@@ -316,7 +316,9 @@ block add-new-hero-via-detail-comp
   The user can *delete* an existing hero by clicking a delete button next to the hero's name.
   Add the following to the heroes component HTML right after the hero name in the repeated `<li>` tag:
 +makeExample('app/heroes.component.html', 'delete')
-
+:marked
+  Add the following CSS to the bottom of `heroes.component.css`.
```

Comment:
Oh, I just noticed this. We need to make this sentence work for all language versions. I'd suggest going with:
> Add the following to the bottom of the `HeroesComponent` CSS file:
> +makeExcerpt('app/heroes.component.css', 'additions')

(Drop the last `''` argument to `makeExcerpt`.)